### PR TITLE
add more tooltips for sidebar items

### PR DESCRIPTION
**Pull request by mohamedamer453**
### Description

Add more tooltips for all sidebar items

image

image

image

### Changes

- Added tooltips for all the items that are available in the sidebar.

### Related Issues

https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1923

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/App.vue [hunk 1]
```diff
@@ -257,40 +257,56 @@ const routes: AppRoute[] = [
...
         title: "Dedicated Machines",
         icon: "mdi-resistor-nodes",
         route: "/deploy/dedicated-nodes",
+        tooltip: "Explore dedicated machines available on the ThreeFold grid.",
+      },
+      {
+        title: "Applications",
+        icon: "mdi-lightbulb-on-outline",
+        route: "/deploy/applications",
+        tooltip: "Deploy ready community solutions.",
```
Comment (AlaaElattar) [inline packages/playground/src/App.vue:291]: should we call it `solutions` or `applications` ?

File: packages/playground/src/App.vue [hunk 2]
```diff
@@ -325,19 +348,31 @@ const routes: AppRoute[] = [
...
+        title: "TF DAO",
+        icon: "mdi-note-check-outline",
+        route: "/tf-chain/dao",
+        tooltip: "Check and vote on DAO propsals.",
+      },
+      {
+        title: "TF Token Bridge",
+        icon: "mdi-swap-horizontal",
+        route: "/tf-chain/bridge",
+        tooltip: "Transfer TFTs to and from stellar.",
```
Comment (AlaaElattar) [inline packages/playground/src/App.vue:363]: you shouldn't specify stellar as there could be other bridges

File: packages/playground/src/App.vue [hunk 3]
```diff
@@ -257,40 +257,56 @@ const routes: AppRoute[] = [
...
         title: "Dedicated Machines",
         icon: "mdi-resistor-nodes",
         route: "/deploy/dedicated-nodes",
+        tooltip: "Explore dedicated machines available on the ThreeFold grid.",
+      },
+      {
+        title: "Applications",
+        icon: "mdi-lightbulb-on-outline",
+        route: "/deploy/applications",
+        tooltip: "Deploy ready community applications.",
```
Comment (xmonader) [inline packages/playground/src/App.vue:291]: I think this needs to be rephrased differently

File: packages/playground/src/App.vue [hunk 4]
```diff
@@ -257,40 +257,56 @@ const routes: AppRoute[] = [
...
       },
       {
         title: "Images",
         icon: "mdi-open-in-new",
         url: "https://hub.grid.tf/",
-        tooltip: "Find or Publish your Flist on 0-Hub",
+        tooltip: "Find or Publish your Flist on 0-Hub.",
       },
       {
         title: "SSHKey",
```
Comment (xmonader) [inline packages/playground/src/App.vue:306]: Change title to SSH Key

File: packages/playground/src/App.vue [hunk 5]
```diff
@@ -325,19 +348,31 @@ const routes: AppRoute[] = [
         title: "Your Profile",
         icon: "mdi-account-supervisor-outline",
         route: "/tf-chain/twin",
+        tooltip: "Check your profile details.",
+      },
+      {
+        title: "TF DAO",
+        icon: "mdi-note-check-outline",
+        route: "/tf-chain/dao",
+        tooltip: "Check and vote on DAO propsals.",
```
Comment (xmonader) [inline packages/playground/src/App.vue:357]: proposals typo

File: packages/playground/src/App.vue [hunk 6]
```diff
@@ -325,19 +348,31 @@ const routes: AppRoute[] = [
...
+        route: "/tf-chain/bridge",
+        tooltip: "Transfer TFTs through different chains.",
       },
-      { title: "TF DAO", icon: "mdi-note-check-outline", route: "/tf-chain/dao" },
-      { title: "TF Token Bridge", icon: "mdi-swap-horizontal", route: "/tf-chain/bridge" },
       {
         title: "TF Token Transfer",
         icon: "mdi-account-arrow-right-outline",
         route: "/tf-chain/transfer",
+        tooltip: "Transfer TFTs through TFChain.",
```
Comment (xmonader) [inline packages/playground/src/App.vue:369]: through -> on

**Review by xmonader — APPROVED** [review]

**Review by AlaaElattar — APPROVED** [review]

**Pull request merged**